Employeemanagement.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace EmployeeManagementt
+{
+    public class Employee
+    {
+        private static int count = 1000;
```

Comment:
why do we need to set the initial value as 1000? can we start it with 0?

Reply:
in the question it was mentioned as
Id, private set | String, autogenerated Emp1000, Emp1001 etc